lib/features/contributor/presentation/cubit/contributor_cubit.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:bloc/bloc.dart';
+import 'package:coffeecard/features/contributor/data/datasources/contributor_repository.dart';
+import 'package:coffeecard/features/contributor/domain/entities/contributor.dart';
+import 'package:equatable/equatable.dart';
+
+part 'contributor_state.dart';
+
+class ContributorCubit extends Cubit<ContributorState> {
+  final ContributorRepository repository;
+
+  ContributorCubit({required this.repository})
+      : super(const ContributorLoaded([]));
```

Comment:
I know we discussed this once, but I'd like to contest this initialisation of the cubit state. Initially, it should *not* have the Loaded state, but rather some Initial state. Having the Loaded[] state has potential to confuse during development or debugging.

Reply:
what do you suggest doing instead?

Reply:
Add a ContributorInitial state